docs/developers/guides/run-a-node/use-binary.mdx

```diff
@@ -56,6 +56,19 @@ Download the genesis file and Besu configuration file.
 
 </Tabs>
 
+:::warning[Important: enable linea-sequencer]
+
+You must install and enable the `linea-sequencer` Besu plugin if you are offering RPC services or 
+hosting nodes for usage. Enabling the plugin is a requirement for using certain endpoints, such as 
+[`linea_estimateGas`](/developers/reference/api/linea-estimategas).
+
+Please see our instructions [here](./linea-sequencer) for installing the plugin.
+
+If you are simply running a node to follow the chain and its activity, rather than using it to 
+submit transactions, you do not need to install `linea-sequencer`.
```

Comment:
```suggestion
If you are running a node to follow the chain activity, rather than 
submit transactions, you don't need to install `linea-sequencer`.
```